components/org.wso2.carbon.identity.auth.valve/pom.xml

```diff
@@ -218,7 +222,13 @@
                             org.wso2.carbon.identity.core.util.*; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.*; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.auth.service.*;version="${org.wso2.carbon.identity.auth.service.version.range}",
-                            org.apache.catalina.*;version="1.7.0"
+                            org.apache.catalina.*;version="1.7.0",
+                            org.wso2.carbon.identity.oauth.common.exception;version="${org.wso2.carbon.identity.oauth.import.version.range}",
+                            org.wso2.carbon.identity.oauth.common;version="${org.wso2.carbon.identity.oauth.import.version.range}",
```

Comment:
Why do we need both org.wso2.carbon.identity.oauth.common and org.wso2.carbon.identity.oauth.common.exception ?

Reply:
Tried to avoid importing the entire package unnecessarily as this valve only requires two classes

Reply:
https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/240/files#diff-4b963f95eab8a4f703ce4747c83c47c7cbad093e87ef4f2a2553d7c6d28b3c0eR227 the package is already imported

Reply:
Resolving as per the offline chat